packages/labs/ssr/src/lib/render-value.ts

```diff
@@ -489,7 +512,46 @@ const getTemplateOpcodes = (result: TemplateResult) => {
           ops.push({
             type: 'custom-element-shadow',
           });
+        } else if (
+          !hydratable &&
+          /^(title|textarea|script)$/.test(node.tagName)
+        ) {
+          const dangerous = willExecute(node);
+          console.log(`dangerous: ${dangerous} – node: `, node);
+          // look for mangled parts in the text content
+          for (const child of node.childNodes) {
+            if (!isTextNode(child)) {
+              throw new Error(
+                `Internal error: Unexpected child node inside raw text node, a ${node.tagName} should only contain text nodes, but found a ${node.nodeName} (tagname: ${node.tagName})`
+              );
+            }
+            const text = child.value;
+            const textStart = child.sourceCodeLocation!.startOffset;
+            flushTo(textStart);
+            const markerRegex = new RegExp(marker.replace(/\$/g, '\\$'), 'g');
+            for (const mark of text.matchAll(markerRegex)) {
+              flushTo(textStart + mark.index!);
+              if (dangerous) {
+                throw new Error(
+                  `Found binding inside an executable <script> tag in a server-only template. For security reasons, this is not supported, as it could allow an attacker to execute arbitrary JavaScript.`
```

Comment:
Is there a workaround? `unsafe` directive?
The error should provide a way to provide executable code if you know what you're doing and want to do it.

Reply:
I haven't proven one out. I'd like to do that as a follow up, this PR is too large as it is!

Reply:
I agree that we'll want to provide a workaround or escape hatch here if the current unsafe directives don't work. As an example, some of our scripts that get included on every page are intentionally inlined and we inject them via a pattern like:
```
<script nonce=${...} type="module">
  ${inlineScriptContents}
</script>
```

Reply:
Initial tests are showing that unsafeHTML works out of the box, so once this lands you should be able to write:

```typescript
serverhtml`
  ${unsafeHTML(`
      <script nonce=${...} type="module">
        ${inlineScriptContents}
      </script>`)}
`;
```